samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -486,6 +493,42 @@ private string OptimizeOpenApiSkillJson(string jsonContent, int tokenLimit, Plan
             : string.Format(CultureInfo.InvariantCulture, "JSON response for {0} is too large to be consumed at this time.", lastSkillInvoked);
     }
 
+    private Type GetOpenApiSkillResponseType(ref JsonDocument document, ref string lastSkillInvoked, ref string lastSkillOperationInvoked, ref bool trimSkillResponse)
+    {
+        Type skillResponseType = typeof(PullRequest); // Use a reasonable default response type
```

Comment:
This shouldn't default to GitHub -- all types inherit from object
```suggestion
        Type skillResponseType = typeof(object); // Use a reasonable default response type
```